local-running/core.local.params.yaml

```diff
@@ -339,6 +351,7 @@ core:
     parseVcClasses: true
     p1JourneysEnabled: false
     sqsAsync: true
+    cimitApiGatewayEnabled: true
```

Comment:
is there a reason why this is set to true by default for local-running?

Reply:
Local running doesn't work without it - if you try and use the Lambdas then you need AWS permissions to invoke them (perfect timing for your PR 😉)